README.md

```diff
@@ -3,3 +3,24 @@
 ## Markdown
 
 With markdown syntaxes you can writ texts in a formatted way. For example headlines, lists, etc. It uses specific characters to format the text or part of the text in a certain way.
+
+### Examples
+
+| Element                              | Markdown Synatx                         |
+| ------------------------------------ | --------------------------------------- |
+| Level 1 headline                     | `# Level 1 headline`                    |
+| Level 2 healdine                     | `## Level 2 headline`                   |
+| Level 5 headline                     | `##### Level 5 headline`                |
+| list item                            | `- list item`                           |
+| [ ] done                             | `[ ] checkbox`                          |
```

Comment:
```suggestion
| [ ] to do                             | `[ ] checkbox`                          |
```
I prefer to do because it isn't marked yet. Just a suggestion.